### ISSUE TITLE
typo in prometheus dnsdist_frontend_tlshandshakefailures

### ISSUE THREAD

**Issue by appliedprivacy** [PowerDNS/pdns#9728]
- Program: dnsdist
 - Issue type: Bug report

### Short description

Typo in prometheus metrics:

is:
```
dnsdist_frontend_tlshandshakefailures{frontend="0.0.0.0:443",proto="TCP (DNS over HTTPS)",thread="0",error="unsupportedProtocol{"} 3
```

should be:
```
dnsdist_frontend_tlshandshakefailures{frontend="0.0.0.0:443",proto="TCP (DNS over HTTPS)",thread="0",error="unsupportedProtocol"} 3
```

### Environment
 - Operating system: FreeBSD
 - Software version: 1.5.1
 - Software source: FreeBSD ports

### Steps to reproduce

enable DoH and webserver() and fetch metrics from /metrics

### Expected behaviour

should be:
```
dnsdist_frontend_tlshandshakefailures{frontend="0.0.0.0:443",proto="TCP (DNS over HTTPS)",thread="0",error="unsupportedProtocol"} 3
```

### Actual behaviour

```
dnsdist_frontend_tlshandshakefailures{frontend="0.0.0.0:443",proto="TCP (DNS over HTTPS)",thread="0",error="unsupportedProtocol{"} 3
```

### Other information

https://github.com/PowerDNS/pdns/blob/6eb9e49ec741c04e4f709a167201fb23a38d81df/pdns/dnsdist-web.cc#L537

**Comment by Habbie** [comment]
Hello! If you go to the file you already linked, there's an edit button, which makes it easy to create a patch and submit it right in the GitHub webinterface. If you don't feel like it, we'll fix it soon. Thanks!

**Comment by rgacogne** [comment]
Fixed by #9729.
